src/learning-catalogue/model/module.ts

```diff
@@ -38,16 +37,17 @@ export class Module {
 
 	formattedDuration: string
 
-	price?: number
+	@IsOptional({
```

Comment:
Should this be moved from to FaceToFaceModule?

Reply:
same comment as above ^^